test/TestCommandLineInvocationSameAsConfig.py

```diff
@@ -1,82 +1,104 @@
 import unittest
-from subprocess import Popen, PIPE
 import os
 import shutil
-import yaml
+
+from ansiblelint import cli
 
 
 class TestCommandLineInvocationSameAsConfig(unittest.TestCase):
+
+    BASE_ARGUMENTS = ['../test/skiptasks.yml']
+
     def setUp(self):
         if os.path.exists(".sandbox"):
             shutil.rmtree(".sandbox")
 
         os.makedirs(".sandbox/subdir")
 
-    def run_ansible_lint(self, args=False, config=None):
-        command = "cd .sandbox; ../bin/ansible-lint ../test/skiptasks.yml"
-        if args:
-            command += " " + args
-
-        if config:
-            with open(".sandbox/.ansible-lint", "w") as outfile:
-                yaml.dump(config, outfile, default_flow_style=False)
-
-        result, err = Popen(
-            [command],
-            stdin=PIPE,
-            stdout=PIPE,
-            stderr=PIPE,
-            shell=True
-        ).communicate()
+    def assert_config_equal(self, args, config):
+        command = self.BASE_ARGUMENTS + args
+        options, _ = cli.get_cli_parser().parse_args(command)
 
-        self.assertFalse(err, "Expected no error but was " + str(err))
+        file_config = cli.load_config(config)
 
-        return result
-
-    def assert_config_for(self, cli_arg, config):
-        with_arg = self.run_ansible_lint(args=cli_arg)
-        with_config = self.run_ansible_lint(config=config)
-
-        self.assertEqual(with_arg, with_config)
+        for key in file_config.keys():
+            self.assertEqual(file_config[key], getattr(options, key))
```

Comment:
We're now using pytest as a test runner, so `assert` statements should be used instead:
```suggestion
            assert file_config[key] == getattr(options, key)
```